crates/store/re_types/definitions/rerun/archetypes/transform3d.fbs

```diff
@@ -40,6 +40,15 @@ table Transform3D (
   /// Specifies the relation this transform establishes between this entity and its parent.
   relation: rerun.components.TransformRelation ("attr.rerun.component_optional", nullable, order: 1600);
 
+  /// Optionally flags the transform as invalid.
```

Comment:
Need something like: "If this is set to true, then all other components on the Transform will be ignored"